platforms/platform-alexa/src/output/AskForPermissionOutput.ts

```diff
@@ -0,0 +1,40 @@
+import { BaseOutput, Output, OutputOptions, OutputTemplate } from '@jovotech/framework';
+
+export type PermissionScope =
+  | 'alexa::alerts:reminders:skill:readwrite'
+  | 'alexa::alerts:timers:skill:readwrite'
+  | string;
+
+export interface AskForPermissionOutputOptions extends OutputOptions {
+  token?: string;
+  permissions?: PermissionScope;
+}
+
+@Output()
+export class AskForPermissionOutput extends BaseOutput<AskForPermissionOutputOptions> {
```

Comment:
my fault, I think `AskForPermissionOutput` is correct since it's a string and not an array. But maybe the option should then be called `permission` and not `permissions`?

Reply:
Thank you for the update. Can you update the docs accordingly?